debug/debug.go

```diff
@@ -293,7 +302,7 @@ func DumpPem(bs []byte, types string, wrt *os.File) error {
 			return fmt.Errorf("could not write PEM: %w", err2)
 		}
 
-		return io.EOF
+		return nil
```

Comment:
Can you please explain this change? Why io.EOF is not ok anymore?

Reply:
I wanted to move away from `io.EOF`.  It had the advantage that it simulated the error response from a reader of the PEM data, but looked counter-intuitive to me.  So I finally moved to `nil` having the opinion was clearer.